### ISSUE TITLE
file upload error to printer da vinci 1.0 aio

### ISSUE THREAD

**Issue by orcema** [reality-boy/miniMover#4]
### Expected Behavior
should upload the file on the printer
### Actual Behavior
A lot of debug window notification were generated and disappeared immediately 

### Steps to reproduce
upload the file to the printer
[supportAntenneGCODE.zip](https://github.com/reality-boy/miniMover/files/2315954/supportAntenneGCODE.zip)
this file was converted with miniMover from this gcode file, generated with slic3R.
[supportAntenne.zip](https://github.com/reality-boy/miniMover/files/2315960/supportAntenne.zip)

### Machine model
da vinci 1.0 aio

the debug output of  Microsoft visual studio 2017 is here below.
A lot of debug window notification were generated and disappeared immediately 

> miniMoverUI.exe' (Win32): Loaded 'D:\Downloads\miniMover-master\miniMover-master\Debug\miniMoverUI.exe'. Symbols loaded.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ntdll.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\kernel32.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\KernelBase.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\user32.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\win32u.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\gdi32.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\gdi32full.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\msvcp_win.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ucrtbase.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\comdlg32.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\msvcrt.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\combase.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\rpcrt4.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\sspicli.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\cryptbase.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\bcryptprimitives.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\sechost.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\SHCore.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\shlwapi.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\shell32.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\WinSxS\x86_microsoft.windows.common-controls_6595b64144ccf1df_6.0.17134.228_none_42f2d07044deda9a\comctl32.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\cfgmgr32.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\windows.storage.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\advapi32.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\kernel.appcore.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\profapi.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\powrprof.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\fltLib.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\setupapi.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ws2_32.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\winmm.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\wlanapi.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\winmmbase.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\winmmbase.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\winmmbase.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\winmmbase.dll'
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\winmmbase.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\imm32.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\uxtheme.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\msctf.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\oleaut32.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\dwmapi.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\devobj.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\wintrust.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\msasn1.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\crypt32.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\TextInputFramework.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\CoreMessaging.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\CoreUIComponents.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ntmarta.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\WinTypes.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\WinTypes.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\WinTypes.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\oleacc.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ole32.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\clbcatq.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Program Files (x86)\Common Files\microsoft shared\ink\tiptsf.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\atlthunk.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\edputil.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ExplorerFrame.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\ExplorerFrame.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ExplorerFrame.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\dui70.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\duser.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\WindowsCodecs.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\thumbcache.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\policymanager.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\msvcp110_win.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\DataExchange.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\dcomp.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\d3d11.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\dxgi.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\dxgi.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\dxgi.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\twinapi.appcore.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\rmclient.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\msftedit.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\msftedit.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\msftedit.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\Windows.Globalization.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\Windows.Globalization.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\Windows.Globalization.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\BCP47mrm.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\BCP47Langs.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\globinputhost.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\propsys.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\xmllite.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\xmllite.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\xmllite.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\UIAutomationCore.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\sxs.dll'. Cannot find or open the PDB file.
mincore\com\oleaut32\dispatch\ups.cpp(2125)\OLEAUT32.dll!74AB69CF: (caller: 74AB656E) ReturnHr(1) tid(1ee4) 8002801D Library not registered.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\StructuredQuery.dll'. Cannot find or open the PDB file.
The thread 0x3a84 has exited with code 0 (0x0).
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\Windows.Storage.Search.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\cldapi.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\cldapi.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\cldapi.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\IconCodecService.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\OneCoreUAPCommonProxyStub.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\OneCoreUAPCommonProxyStub.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\OneCoreUAPCommonProxyStub.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\twinapi.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\twinapi.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\twinapi.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\mpr.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\drprov.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\drprov.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\drprov.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\winsta.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ntlanman.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\davclnt.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\davhlpr.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\wkscli.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\bcrypt.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\cscapi.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\netutils.dll'. Cannot find or open the PDB file.
Exception thrown at 0x759DDDC2 (KernelBase.dll) in miniMoverUI.exe: 0x000006BA: The RPC server is unavailable.
The thread 0x3aa4 has exited with code 0 (0x0).
The thread 0x183c has exited with code 0 (0x0).
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\coml2.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\linkinfo.dll'. Cannot find or open the PDB file.
The thread 0x23e4 has exited with code 0 (0x0).
The thread 0x32c4 has exited with code 0 (0x0).
The thread 0x1c9c has exited with code 0 (0x0).
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\Windows.StateRepositoryPS.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\Windows.StateRepositoryPS.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\Windows.StateRepositoryPS.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\apphelp.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\networkexplorer.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\networkexplorer.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\networkexplorer.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\networkitemfactory.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\dtsh.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\FirewallAPI.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\fwbase.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\fwpolicyiomgr.dll'. Cannot find or open the PDB file.
The thread 0x3768 has exited with code 0 (0x0).
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\npmproxy.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\fundisc.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\fundisc.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\fundisc.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\fdProxy.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\actxprxy.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\fdWCN.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Users\orce\AppData\Local\Microsoft\OneDrive\18.131.0701.0007\FileSyncShell.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\ktmw32.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\version.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\wininet.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\WcnApi.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\WcnApi.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\WcnApi.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\fdWNet.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\IPHLPAPI.DLL'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\atl.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\dfscli.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\dfscli.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\dfscli.dll'. Cannot find or open the PDB file.
Exception thrown at 0x759DDDC2 (KernelBase.dll) in miniMoverUI.exe: 0x000006BA: The RPC server is unavailable.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\browcli.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\OneCoreCommonProxyStub.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\OneCoreCommonProxyStub.dll'
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\OneCoreCommonProxyStub.dll'. Cannot find or open the PDB file.
The thread 0x1e54 has exited with code 0 (0x0).
The thread 0xda8 has exited with code 0 (0x0).
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\mswsock.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\dnsapi.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\nsi.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\rasadhlp.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\FWPUCLNT.DLL'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\dhcpcsvc6.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Loaded 'C:\Windows\SysWOW64\dhcpcsvc.dll'. Cannot find or open the PDB file.
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\msftedit.dll'
The thread 0x22ec has exited with code 0 (0x0).
The thread 0x1448 has exited with code 0 (0x0).
'miniMoverUI.exe' (Win32): Unloaded 'C:\Windows\SysWOW64\twinapi.dll'
'miniMoverUI.exe' (Win32): Unloaded 'C:\Program Files (x86)\Common Files\microsoft shared\ink\tiptsf.dll'
The thread 0x750 has exited with code 0 (0x0).
The thread 0x3844 has exited with code 0 (0x0).
The thread 0x1958 has exited with code 0 (0x0).
The thread 0x3948 has exited with code 0 (0x0).
The thread 0x98c has exited with code 0 (0x0).
The thread 0x3d4 has exited with code 0 (0x0).
The thread 0x1818 has exited with code 0 (0x0).
The thread 0x3bc0 has exited with code 0 (0x0).
The thread 0x2890 has exited with code 0 (0x0).
The thread 0x8e0 has exited with code 0 (0x0).
The thread 0x2728 has exited with code 0 (0x0).
The thread 0x361c has exited with code 0 (0x0).
The thread 0x115c has exited with code 0 (0x0).
The thread 0xe7c has exited with code 0 (0x0).
The thread 0x568 has exited with code 0 (0x0).
The thread 0x3520 has exited with code 0 (0x0).
The thread 0x3a88 has exited with code 0 (0x0).
The thread 0x3ba4 has exited with code 0 (0x0).
The thread 0xdc4 has exited with code 0 (0x0).
The thread 0x3540 has exited with code 0 (0x0).
The thread 0x170c has exited with code 0 (0x0).
The thread 0x1fcc has exited with code 0 (0x0).
The thread 0x12d4 has exited with code 0 (0x0).
The thread 0x1118 has exited with code 0 (0x0).
The thread 0x3c4 has exited with code 0 (0x0).
The program '[15260] miniMoverUI.exe' has exited with code 0 (0x0).

**Comment by reality-boy** [comment]
Sorry you are having troubles.  So I tried to replicate your issues but could not.  First off I don't have a 1.0 AIO or really any of the 1.x or 2.x machines available to test with, so that limits what I can do from here.  I was able to convert the gcode to a 3w that targeted a 1.0 AIO machine and load the resulting file into XYZWare.  And I can print the gcode to my miniMaker, although it failed to print because the model is too large for my machine.

With that said, I don't currently support printing to a 1.x machine because they talk using a different protocol than the more recent printers.  All that works with your printer is to convert form a gcode file to a 3w and back again.  That should be enough for you to get data from your slicer into XYZWare.

So basically you want to just use the convert file button and target dropdown.

**Comment by reality-boy** [comment]
If you have the time, could I get you to help figure out where the crash is coming from?  For starters when does it crash, at load time or after hitting a particular button (say print). 

Can you create a text file named debug.txt and place it in the same directory as the minimoverui.exe executable then try to do whatever your doing to generate the crash.  If you can post that debug output here it may help me pin down what is going on.

**Comment by orcema** [comment]
The error happens when i click print. The same error appears regardless my printer is connected to my laptop or not.
Please find in attachment the requested file 
[debug.txt](https://github.com/reality-boy/miniMover/files/2333307/debug.txt)

**Comment by reality-boy** [comment]
Thanks for that, it is a big help, I will try to track down the crash.  

For now just don't hit print, actually I currently don't support printing to this printer at all, stick to the convert button.

**Comment by reality-boy** [comment]
As far as I know this is fixed, well sort of.  I don't support printing to an older 1.0 printer, but I don't crash any more either.  I don't know if I will ever support a 1.0 printer, but that would be a new feature and not a bug.
